pjmedia/src/pjmedia/vid_stream.c

```diff
@@ -1612,8 +1612,8 @@ PJ_DEF(pj_status_t) pjmedia_vid_stream_get_port(pjmedia_vid_stream *stream,
 {
     pjmedia_stream_common *c_strm = (stream? &stream->base: NULL);
```

Comment:
This init line looks a bit odd now, perhaps just init after the assertion check?